OJP/OJP_JourneySupport.xsd

```diff
@@ -644,7 +644,7 @@
 			<xs:choice minOccurs="0">
 				<xs:element name="NumberOfResults" type="xs:positiveInteger" minOccurs="0">
 					<xs:annotation>
-						<xs:documentation>The number of trip results that the user wants to see at least (TRIP REQUEST POLICY.maximumNumberofJourneys). Be aware that some implementations will deliver one of the TripResults before the indicated departure time. This means one always must check the results against the parameters before using them.</xs:documentation>
+						<xs:documentation>The number of trip results that the user wants to see at least (TRIP REQUEST POLICY.maximumNumberofJourneys). Be aware that some implementations will deliver one of the TripResults before the indicated departure time. MultiPointTripRequests may deliver more results than requested depending on the MultiPointType setting. This means one always must check the results against the parameters before using them.</xs:documentation>
```

Comment:
What does the *TRIP REQUEST POLICY.maximumNumberofJourneys* refer to? I don't find such a thing in OJP. And it is at odds with "that the user wants to see at *least*".